bin/minify.js

```diff
@@ -89,13 +91,52 @@ function logAll(array) {
 
 function help() {
     const bin = require('../help');
-    const usage = 'Usage: minify [options]';
     
-    console.log(usage);
+    console.log('Usage:');
+    console.log('  minify [options]');
+    console.log('    This will read from stdin and output to stdout.');
+    console.log('  minify filePath [options]');
+    console.log('    This will read from a file and output to stdout.');
+    console.log('');
+    console.log('Examples:');
+    console.log('  minify -h');
+    console.log('    Print this message.');
+    console.log('');
+    console.log('  cat hello.js > minify --js > hello.min.js');
+    console.log('    Minify js as part of a pipeline of stream processors.');
+    console.log('');
+    console.log('  minify hello.html --htmlOptions="{\\"removeOptionalTags\\": false}" > hello.min.html');
+    console.log('    Minify an html file, writing it to a new file, without removing optional tags.');
+    console.log('');
```

Comment:
Why do you need so many logs here? Why don`t you used loop?

Reply:
in 5ea30b5 I have:
changed this to a single multiline string, so it requires only one console.log statement